internal/cmd/schema-registry/command_schema_create.go

```diff
@@ -103,7 +109,31 @@ func (c *command) schemaCreate(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	response, _, err := srClient.DefaultApi.Register(ctx, subject, srsdk.RegisterSchemaRequest{Schema: string(schema), SchemaType: schemaType, References: refs})
+	metadata, err := ReadMetadata("metadata", cmd)
+	if err != nil {
+		return err
+	}
+
+	ruleset, err := ReadRuleSet("rule-set", cmd)
+	if err != nil {
+		return err
+	}
+
+	normalize, err := cmd.Flags().GetBool("normalize")
+	registerOpts := srsdk.RegisterOpts{
+		Normalize: optional.NewBool(normalize),
+	}
+	if err != nil {
+		return err
+	}
+
+	response, _, err := srClient.DefaultApi.Register(ctx, subject,
+		srsdk.RegisterSchemaRequest{Schema: string(schema),
```

Comment:
Please don't initialize multi-line structs in a function call. It's hard to read this way.